key/docs/README.md

```diff
@@ -88,41 +88,41 @@ _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormati
 
 #### KeyUsage
 
-Determines the cryptographic operations for which you can use the CMK. The default value is ENCRYPT_DECRYPT. This property is required only for asymmetric CMKs. You can't change the KeyUsage value after the CMK is created.
+Determines the cryptographic operations for which you can use the AWS KMS Key. The default value is ENCRYPT_DECRYPT. This property is required only for asymmetric AWS KMS Keys. You can't change the KeyUsage value after the AWS KMS Key is created.
 
 _Required_: No
 
 _Type_: String
 
 _Allowed Values_: <code>ENCRYPT_DECRYPT</code> | <code>SIGN_VERIFY</code>
 
-_Update requires_: [Replacement](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement)
+_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Comment:
This gets generated incorrectly by the CFN tooling. We need to manually change it back to "Replacement".